routes/onGuess.go

```diff
@@ -0,0 +1,60 @@
+package routes
+
+import (
+	i "johnSamilin/golang-hangman/interfaces"
+	"johnSamilin/golang-hangman/utils"
+	"net/http"
+	"strings"
+)
+
+// handler for guessing. request body must contain { letter: 'j' } structure
+func MakeGuess(state *i.GameState, req *http.Request) (int, string) {
+	var clientId = req.Header.Get("X-Client-Id")
+
+	jsonResult := i.GuessRequest{}
+	var err = utils.GetDataFromBody(req.Body, jsonResult)
+	if err != nil {
+		return 500, "Unable to parse request"
+	}
+
+	var word = state.Clients[clientId].Word
```

Comment:
do not use `var ...` without a real reason. if you avoid the type, use the shortcut (`word := ...`).